src/script/auth/module/action/AuthAction.ts

```diff
@@ -337,7 +337,7 @@ export class AuthAction {
         }
         const clientType = persist ? ClientType.PERMANENT : ClientType.TEMPORARY;
 
-        await core.init(clientType, {initClient: false});
+        await core.init(clientType);
```

Comment:
This `init` method now never init the client, it has to be called in a second step (which is a more natural way of using the api)